helper/schema/schema_test.go

```diff
@@ -439,8 +439,9 @@ func TestSchemaMap_Diff(t *testing.T) {
 			Diff: &terraform.InstanceDiff{
 				Attributes: map[string]*terraform.ResourceAttrDiff{
 					"availability_zone": &terraform.ResourceAttrDiff{
-						Old: "",
-						New: "${var.foo}",
+						Old:         "",
+						New:         "${var.foo}",
+						NewComputed: true,
```

Comment:
This here is an actual test CHANGE versus a new change. The old behavior I believe to be incorrect since there was no way to tell that when applying, the EXACT value of `${var.foo}` wasn't expected... the diff comparison would simply never work.